### PR TITLE
EVA-1438 - Automate the triggering and checking of a dbSNP assembly import

### PR DESCRIPTION
**Pull request by sundarvenkata-EBI**
I have not been able to execute the scripts yet. But I am submitting this PR for initial review.

### CODE REVIEW

File: eva-accession-import-automation/run_accession_import.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import os
+import argparse
+import sys
+import json
+import subprocess
+from __init__ import *
+
+
+def run_command(command):
```
Comment (jmmut) [inline eva-accession-import-automation/run_accession_import.py]: I guess this function is not used to test the commands printed first, right?

File: eva-accession-import-automation/run_accession_import.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                                                   "-f {fasta_file_path} " +
+                                                   "-d {metadb} -u {metauser} -h {metahost} " +
+                                                   "-D {job_tracker_db} -H {job_tracker_host} " +
+                                                   "--mongo-acc-db {mongo_acc_db} --mongo-auth-db {mongo_auth_db} " +
+                                                   "--mongo-user {mongo_user} --mongo-password {mongo_password} " +
+                                                   "--mongo-host {mongo_host} --mongo-port {mongo_port}")\
+                                                .format(**program_args)
+
+    program_args["properties_file_path"] = os.path.sep.join(["{species_accessioning_import_folder}",
+                                                             "{assembly_accession}_b{build}.properties"]
```
[the remark this answers is not in the document]
Comment (jmmut) [reply eva-accession-import-automation/run_accession_import.py]: done

File: eva-accession-import-automation/README.md [hunk 3]
```diff
@@ -4,6 +4,36 @@
     pip install -r requirements.txt
     ```
 # Usage
+## Run the automation script
+* Example
+    ```bash
+    python3 run_accession_import.py -s chickpea_3827 --scientific-name cicer_arietinum -a GCA_000331145.1 -b 146 -l -n ASM33114v1 -p /path/to/private-dev-config.json
+    ```
+    * Private configuration file example
```
Comment (andresfsilva) [inline eva-accession-import-automation/README.md]: `genbank_equivalents_file` needs to be included here as well

**Comment by sundarvenkata-EBI** [comment]
Will perform further work on this as discussed in today's Sprint meeting.

**Pull request merged**